components/atoms/MoreInfo.js

```diff
@@ -0,0 +1,42 @@
+import Link from "next/link";
+import PropTypes from "prop-types";
+import { faArrowAltCircleRight } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+/**
+ *  More Information Component
+ */
+
+export default function MoreInfo(props) {
+  const googleSearch = "https://google.ca/search?q="
+    .concat(props.text)
+    .replace(/\s/g, "+");
+
+  return (
+    <div className="pb-2 pt-2">
+      <div className="pb-2 pt-2 px-3 border border-custom-blue-dark rounded text-custom-blue-dark cursor-pointer hover:bg-gray-light-report focus:ring-1 focus:ring-black focus:ring-offset-1 active:bg-gray-active">
+        <Link href={googleSearch}>
+          <a className="block w-full" id={props.id}>
+            <h6>
```

Comment:
not sure about the H6 here, it implies the page will have h2-h5

Reply:
I was following what the Figma designs had but this is a fair point. I removed the h6 tag and instead added styling to make it look like an h6.